lib/provider/bpmn/parts/NameProps.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 var nameEntryFactory = require('./implementation/Name'),
-    is = require('bpmn-js/lib/util/ModelUtil').is;
+    is = require('bpmn-js/lib/util/ModelUtil').is,
+    getBusinessObject = require('bpmn-js/lib/util/ModelUtil').getBusinessObject,
+    cmdHelper = require('../../../helper/CmdHelper');
 
 module.exports = function(group, element, translate) {
 
+  function setGroupName(element, values) {
+    var bo = getBusinessObject(element),
+        categoryValueRef = bo.categoryValueRef || {};
```

Comment:
This is not sufficient to instantiate a missing `categoryValueRef`.

We must 

* properly create the category via the `BpmnFactory`
* add the category in the respective place
* link it to the BO if it does not exist